CHANGES

```diff
@@ -1,13 +1,34 @@
 Changelog
 =========
 
-Version 1.x.x
+Version 1.2.0
 -------------
 
 Unreleased.
 
+*   Drop support for Python 2.7, 3.4-3.6, add support for Python 3.7-3.11.
+
 *   Add type annotations (PEP 484 and PEP 561).
 
+*   Many CI additions and improvements.
+
+*   Include tests in the PyPI tarball.
+
+*   Improve the test coverage.
```

Comment:
A nitpick: usually it makes sense to put CI and testing improvements to the end of the release notes, because they're not very relevant for the end users.